Kernel/Devices/Storage/USB/USBStorageController.h

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2023, Leon Albrecht <leon.a@serenityos.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/IntrusiveList.h>
+#include <Kernel/Devices/Storage/StorageController.h>
+#include <Kernel/Devices/Storage/StorageDevice.h>
+#include <Kernel/Devices/Storage/StorageManagement.h>
+#include <Kernel/Devices/Storage/USB/MassStorageDevice.h>
+
+namespace Kernel::USB {
+
+class StorageController : public ::StorageController {
+public:
+    static StorageController& the();
+
+    virtual LockRefPtr<StorageDevice> device(u32 index) const override
+    {
+        // FIXME: Avoid this const_cast
+        return const_cast<Vector<NonnullLockRefPtr<BulkSCSIInteface>>&>(m_devices).at(index);
+    }
+    virtual size_t devices_count() const override
+    {
+        dbgln("Query USB Storage Device Count");
```

Comment:
This seems a bit useless/spammy

Reply:
Ah missed to remove that one
(Only fires once during initial storage device enumeration though)